website/content/installation/_index.md

```diff
@@ -112,3 +112,23 @@ secretGenerator:
 generatorOptions:
  disableNameSuffixHash: true
 ```
+
+## Upgrade Current Version
+To upgrade current version of metallb to new release, always look at [the release notes](https://metallb.universe.tf/release-notes/) to see the changes and action required.
+
+After that we can install new metallb version by downloading the new manifests and `kubectl apply -f` them. For example we upgrade from metallb v0.8.1 to v0.9.3. By reading [the release notes](https://metallb.universe.tf/release-notes/) we know that newer metallb use new algorithm to detect dead nodes. That algorithm need `secretkey` that we need to create. Here the steps to upgrade metallb v0.8.1 to v0.9.3:
```

Comment:
What if we just mention the generalities, without an example?

```suggestion
After that we can install new metallb version by downloading the new manifests and `kubectl apply -f` them.
 
Take into account the known limitations for [layer2](https://metallb.universe.tf/concepts/layer2/#limitations) and [bgp](https://metallb.universe.tf/concepts/bgp/#limitations) when doing so.
```

Reply:
Ah, I agree. That is good idea to make it more general.

Reply:
Thanks!